src/hpp/corbaserver/rbprm/rbprmbuilder.py

```diff
@@ -41,8 +44,8 @@ def __init__(self, load=True):
     # \param packageName name of the package from where the robot will be loaded
     # \param urdfSuffix optional suffix for the urdf of the robot package
     # \param srdfSuffix optional suffix for the srdf of the robot package
-    def loadModel(self, urdfName, urdfNameroms, rootJointType, meshPackageName, packageName, urdfSuffix, srdfSuffix):
-        Robot.__init__(self, urdfName, rootJointType, False)
+    def loadModel(self, urdfName, urdfNameroms, rootJointType, meshPackageName, packageName, urdfSuffix, srdfSuffix, client):
+        Robot.__init__(self, urdfName, rootJointType, False, client=client)
```

Comment:
@stonneau @pFernbach Any idea why the parent initializer is called in loadModel and not in __init__ ?

Reply:
If there was I don't remember. By the ways  shouldn't `client` have a default value to `None` to keep backward compatibility with the current scripts ?

Reply:
I set client parameter to None by default in loadModel() and loadFullBodyModel() in the last commit.